packages/jest-util/src/isPromise.ts

```diff
@@ -0,0 +1,14 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+// capture global.Promise before it may potentially be overwritten
```

Comment:
Is this needed? We have a babel plugin that already handles this:

https://github.com/facebook/jest/blob/4c259423b40be4a20afa3903d1d40e12469d425a/scripts/babel-plugin-jest-native-globals.js#L14-L16

Reply:
That plugin is not currently applied to `jest-util`. If I add it and remove this declaration, the new e2e test fails because apparently `isPromise` then doesn't consider the user's Promise a Promise, so the general error message is printed.